### ISSUE TITLE
Provide the ability to remember column widths using local storage

### ISSUE THREAD

**Issue by jadrake75** [jadrake75/ng-scrolling-table#65]
At the very least, if you are navigating between routes, the column widths should be remembered in a service.  LocalStorage would be nice on a per/computer basis.

This may only be workable for tables that declare an ID or ref-id since we'll need a constant "key" to cache on.

**Comment by Xanir** [comment]
Should this enhancement be expanded to cover other features such as:
 * column reorder
 * columns hiding

**Comment by jadrake75** [comment]
Yes seems reasonable.  Note: 1.0 is not the "next" release so is not as important to work on yet - but yes.
